sale_order_line_input/models/sale_order.py

```diff
@@ -11,18 +11,40 @@ class SaleOrderLine(models.Model):
         related='order_id.pricelist_id',
         readonly=True,
     )
+    # HACK: Overwrite field because the related fields do not get the default
+    # value correctly when user adds a new record.
+    company_id = fields.Many2one(
```

Comment:
This hack is very conflicting and will affect all the sales orders, normal and input ones. Don't you have any other way (through `default_get` or similar) to get the same goal?